src/main/java/ui/components/pickers/MilestonePickerState.java

```diff
@@ -273,6 +275,12 @@ private void addMatchingMilestones(List<PickerMilestone> milestonesToBePopulated
                 .findFirst();
     }
 
+    private List<PickerMilestone> getSelectableMilestones(List<PickerMilestone> milestones) {
```

Comment:
Will need a header comment here. What is `Selectable` and how is it defined? I didn't understand right away why is it selectable by reading the method body.

If it is difficult to write a header comment that is not tied to implementation, maybe renaming the method works better.